data-pipeline/src/pipeline/config.py

```diff
@@ -1,45 +1,47 @@
 income_category_map = {
     "academies": {
-        "DFE/EFA Revenue grants (includes Coronavirus Government Funding": "Income_Other DFE grants",
```

Comment:
We will need this for the historic incomes though

Reply:
Unless I'm missing something the conditonal mapping in the `process_aar_data` should handle this.